client/comms/wsconn.go

```diff
@@ -298,13 +302,18 @@ func (conn *wsConn) keepAlive(ctx context.Context) {
 				log.Errorf("Reconnect failed: %v", err)
 				conn.reconnects++
 				if conn.reconnects+1 < maxReconnects {
-					conn.queueReconnect()
+					conn.queueReconnect(rcInt)
+					// Increment the wait up to PingWait.
+					if rcInt < conn.cfg.PingWait {
+						rcInt += reconnectInterval
+					}
```

Comment:
Should we cap `rcInt` at maybe a minute or two? `maxReconnects` is 20,000,000, and we want the client to be responsive when the DEX comes back online.